StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkViewController.swift

```diff
@@ -114,14 +118,16 @@ final class PayWithLinkViewController: UINavigationController {
         intent: Intent,
         configuration: PaymentSheet.Configuration,
         shouldOfferApplePay: Bool = false,
-        shouldFinishOnClose: Bool = false
+        shouldFinishOnClose: Bool = false,
+        shouldPayInFlow: Bool = true
```

Comment:
This parameter was originally named when the implementation required more extensive changes to this logic. As is, it's literally just a flag that says whether the confirm button should say "Pay" or "Continue". Should this be renamed and/or inverted to better reflect its departure from default behavior? (The fact that it defaults to `true` when all other flags here default to `false` feels like a sign this needs adjusting.)

Reply:
I'm not particularly bothered by its default value, but the behavior doesn't feel like it matches the description:
>         ///   - shouldPayInFlow: Whether or not Link should process payment within the Link flow

The "process payment" part feels off if it's just controlling a string.

Reply:
Agreed. That is also an artifact of the earlier implementation 😅 Seems reasonable to make it more specific here now that it's only controlling the one string - maybe something like `shouldDisplayPrice`? Or would it be better to let the caller specify the CTA override? (e.g. `customCTA: ConfirmButton.CallToAction? = nil`)

Reply:
In the latest commit, I've opted for that approach (renaming to `callToAction` for clarity), which also enabled some nice cleanup. I was overlooking setup intents, which neither pay in the flow nor display the price on the confirm button.